Product/Production/Adapters/General/CONNECTAdminGUI/pom.xml

```diff
@@ -231,7 +231,7 @@
     <dependency>
       <groupId>commons-fileupload</groupId>
       <artifactId>commons-fileupload</artifactId>
-      <scope>runtime</scope>
+      <version>1.3.2</version>
```

Comment:
We manage version in Product/pom.xml which will cascade down to submodule.  Please consider to revert this change.